modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -883,7 +882,52 @@ Number of active consumers within a consumer group.
 
 *Labels*:
 
-* `group`
+- `redpanda_group`
+- `shard`
+
+---
+
+=== redpanda_kafka_consumer_group_lag_max
+
+Maximum consumer group lag across topic partitions. This metric is useful for identifying the most delayed consumer group.
+
+To enable this metric, you must enable consumer group metrics in your cluster properties. Add the following settings to your Redpanda configuration:
+
+- xref:reference:properties/cluster-properties.adoc#enable_consumer_group_metrics[`enable_consumer_group_metrics`]: A list of properties to enable for consumer group metrics. You must add the `consumer_lag` property to enable consumer group lag metrics.
+- xref:reference:properties/cluster-properties.adoc#consumer_group_lag_collection_interval_sec[`consumer_group_lag_collection_interval_sec`] (optional): The interval in seconds for collecting consumer group lag metrics. The default is 60 seconds.
+
+*Type*: gauge
+
+*Labels*:
+
+- `redpanda_group`
+- `shard`
```

Comment:
[I'm going to eliminate shard for this one](https://github.com/redpanda-data/redpanda/pull/25383)

Reply:
Removed